src/index.html

```
@@ -15,6 +15,7 @@
 
     <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet" />
     <link href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Mono:300" rel="stylesheet" />
+    <script src="assets/env.js"></script>
```

HTMLHint (1.8.0)

[warning] The <script> tag cannot be used in a <head> tag.

(head-script-disabled)